pip_audit/_virtual_env.py

```diff
@@ -106,7 +106,7 @@ def post_setup(self, context: SimpleNamespace) -> None:
             *self._install_args,
         ]
         try:
-            run(package_install_cmd, state=self._state)
+            run(package_install_cmd, log_stdout=True, state=self._state)
```

Comment:
Let's also try and do this with `--dry-run --report`: there's a chance that might be faster than doing a full "normal" install, and the JSON it produces should be easier to consume.

Reply:
Ref: https://github.com/pypa/pip/pull/10771

Reply:
[6bd5a21](https://github.com/pypa/pip-audit/pull/523/commits/6bd5a211d7af322a6e1aeea6f10de69ae27ac278)